frontend/src/main.js

```diff
@@ -169,7 +169,7 @@ analyticsServ.ga().then( (gajson) => {
 
   let vuetifyOpts = {
     icons: {
-      iconfont: 'md', // 'md' || 'mdi' || 'fa' || 'fa4'
+      iconfont: 'mdi', // 'md' || 'mdi' || 'fa' || 'fa4'
```

Comment:
This is a larger scope change and you haven't changed all of the icons across the entire app

Reply:
addressing this per page as I get to them

Reply:
I suggest leaving this change out until they are all done., it shouldn't be too hard to do a change on all the icons either though.

Reply:
Agreed but is already resolved on branch for next ticket, will be resolved with that one